r37465 | rtogni | 2015-08-28 21:37:24 +0200 (Fri, 28 Aug 2015) | 3 lines

Use av_frame_free() instead of av_freep() to free AVFrame

Index: libmpcodecs/vd_ffmpeg.c
===================================================================
--- libmpcodecs/vd_ffmpeg.c	(revision 37464)
+++ libmpcodecs/vd_ffmpeg.c	(revision 37465)
@@ -539,7 +539,7 @@
     }
 
     av_freep(&avctx);
-    av_freep(&ctx->pic);
+    av_frame_free(&ctx->pic);
     free(ctx);
 }
 
